### ISSUE TITLE
SMS Budget

### ISSUE THREAD

**Issue by newmickymousse** [yearn/budget#96]
TLDR; Strategists ms has no income other than what ybudget approves, thus to keep the team running we need a budget renewal. yBudget grants mainly used to support current full-time contributors.

### Scope

In line with YIP-66 (Instruct yBudget to set funds aside for a dedicated team budget for Strategists), we are asking for a new budget for the following quarter. 

In the meantime, there is an on-going effort to restructure all strategist-related work into smaller and more speceialized teams with specific responsibilities (e.g., vault management, dinobots, future yield, among others). These teams will form and submit their own budget requests independently by the end of this proposal period.

An example of how this will work is in how the Vault Management team operates today with a clearly defined team, accountabilities, and a recurring budget request that exists outside of the general strategist grant.

Hence, we are requesting a budget of $125k a month for the strategists team. (actual budget we need is $150k but we are using last quarters’s unused funds to cover the remaining $25k*3=$100k) 

### Plan

After poorcomp, each team needs to submit a proposal to request budget for their operations. Strategist ms is not only formed by full-time contributors but also by several people that we have been training and knows how yearn’s vaults and strategies work. This team has been paid first by the funds collected from the donation of strategists’ rewards (pre-poorcomp) and later through yearn’s treasury with a monthly grant of $200k. Now, we understand that revenue has decreased and that we need to adjust our budget. Thus, our new proposal considers a 37.5% reduction from the previous one.

As mentioned in our previous proposal, we have included mentorships for new strategists and kept improving our strats ops to become more efficient. 

### Deadline

2022-12-31

### People

Current SMS

### Money

Why $150k/mo? (we are only asking for $125k/mo)
150k would cover:

A set of dedicated contributors for the continued effort on maintaining and releasing new vaults and strategies. A set of dedicated strategists team members. The current team costs $120k (a group of around 8 people at a $15k rate/mo)

Strategist grants budget: $20k
 ⁃ The strategist ms has a $20k budget each month to give grants to non-strategists for building of toolings and Strat ops.

Slippage: $10k
 ⁃ Round up for other expenses. Funds for testing experimental strategies, infrastructure (nodes etc), unforseen expenses. Other stuff at the discretion of the strategist team.

### Amount

375000

### Wallet address

0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7

### Reporting

- [X] I agree to post weekly updates

**Comment by banteg** [comment]
since the vault management team has already spun out in #93, the actual reduction is 29% not 37.5%.

this seems insufficient to me given much more aggressive reductions across the board. for example, the stablecoin grants for full-time devs [have been cut](https://gov.yearn.finance/t/ybudget-update-bear-minimum-comp/12609) by 83.8%.

would love to see a detailed breakdown of how the previous allocation was used before making any calls on this one.

**Comment by newmickymousse** [comment]
explained here: https://hackmd.io/CDe6F03ZSDON_AEM6eDKnw?view
